assets/css/index.css

```diff
@@ -1,10 +1,44 @@
-#ProjectView {
-	width: 100%;
-
+body { 
+	height:100%;
+	background-color: rgba(125, 212, 250, 1);	
 }
 
 .row {
-	margin:4px;
-	padding: 8px;
 	min-height:40px;
+}
+
+#HistoryView {
+	margin-top: 50px; /* because of the header */ 
+	height:10%;
+	min-height: 60px;
+	padding:4px;
+	background-color: rgba(125, 212, 250, 1);
+}
+
+#HistorySketchesView {
+	overflow-x:scroll; 
```

Comment:
Hmm, but we could have the scrollbar only appear when needed.

I actually think that showing it all the time like you have here is best, as long as we theme it so that it's not visually distracting.